docs/platform-specification.md

```diff
@@ -839,8 +839,9 @@ custom menu we are going to create and must be declared at the beginning of the
     menu.cpu=Processor
     [.....]
 
-in this case, the menu name is "Processor".<br> Now let's add, always in the boards.txt file, the default configuration
-(common to all processors) for the duemilanove board:
+in this case, the menu name is "Processor". This step is really important, if a board defines an option but the name of
+that option is not defined the board won't be loaded. <br> Now let's add, always in the boards.txt file, the default
+configuration (common to all processors) for the duemilanove board:
```

Comment:
```suggestion
in this case, the menu name is "Processor".<br> Now let's add, always in the boards.txt file, the default configuration
(common to all processors) for the duemilanove board:
```
My opinion is that the added text is not necessary. The entire document is filled with properties that must be defined in order for the platform to work correctly. Adding an admonitory sentence for each would make the document overly verbose and I don't think this is a common enough problem to justify singling this specific property out for special treatment.

The "must" used in the sentence introducing this property:
> We must first define a set of menu.MENU_ID=Text properties.

is already a standard technical term for indicating requirement levels (e.g., https://datatracker.ietf.org/doc/html/rfc2119) so I think its mandatory nature is already well specified for the intended audience.

I am interested in the idea of a standardized non-prose system for indicating the required/optional nature of components in the specifications. The approach taken for [the library.properties specification](https://arduino.github.io/arduino-cli/dev/library-specification/#libraryproperties-file-format) was saying "Unless noted otherwise below, all fields are required." in the introduction and then appending `(optional)` to the optional field names.

Reply:
Ok for me. 👍